### PR TITLE
PREAPPS-2704: Local Storage | Implement permanent delete (delete an email from local trash)

### PR DESCRIPTION
**Pull request by sachinpatel88**

### CODE REVIEW

File: src/schema/schema.ts [hunk 1]
```diff
@@ -321,7 +321,13 @@ export function createZimbraSchema(
 
 				changeFolderColor: (_, variables) =>
 					client.changeFolderColor(variables as FolderActionChangeColorInput),
-				folderAction: (_, { action }) => client.folderAction(action),
+				folderAction: (_, { action, context }) => {
```
Comment (itsmeprashant) [inline src/schema/schema.ts]: `folderAction: (_, { action, context }) => context && context.local ? localStoreClient.action() : client.folderAction(action),`

File: src/schema/schema.ts [hunk 2]
```diff
@@ -321,7 +321,10 @@ export function createZimbraSchema(
 
 				changeFolderColor: (_, variables) =>
 					client.changeFolderColor(variables as FolderActionChangeColorInput),
-				folderAction: (_, { action }) => client.folderAction(action),
+				folderAction: (_, { action, context }) =>
```
Comment (yashrajpchavda) [inline src/schema/schema.ts]: if I am not mistaken, actually `context` comes as a second parameter in the resolver function, previously, have we done anything different for folder action mutation to make `context` part of `variables` itself?

Comment (yashrajpchavda) [reply src/schema/schema.ts]: Sorry, I missed it earlier, but for other folder related actions (for local as well as remote), we are using `action` mutation from this file, does the remote call to permanently delete a folder, use this method?

Comment (sachinpatel88) [reply src/schema/schema.ts]: Yeah It's not being used. I think I need to delete this branch. :D

**Comment by sachinpatel88** [comment]
Delete as folderAction mutation changes not required.

**Pull request closed without merging**